snomed/com.b2international.snowowl.snomed.exporter.server/src/com/b2international/snowowl/snomed/exporter/server/dsv/MapTypeRefSetDSVExporter.java

```diff
@@ -100,9 +102,11 @@ public File executeDSVExport(final OMMonitor monitor) throws SnowowlServiceExcep
 		final IEventBus bus = applicationContext.getService(IEventBus.class);
 		
 		final SnomedReferenceSet refSet = SnomedRequests.prepareGetReferenceSet(exportSetting.getRefSetId())
-				.setExpand("members(limit:" + Integer.MAX_VALUE + ", expand(referencedComponent(expand(pt()))))")
+				.setExpand("members(limit:" + Integer.MAX_VALUE + ", expand(referencedComponent(expand(preferredDescriptions()))))")
 				.setLocales(languageSetting.getLanguagePreference())
-				.build(SnomedDatastoreActivator.REPOSITORY_UUID, branchPath.getPath()).execute(bus).getSync();
+				.build(SnomedDatastoreActivator.REPOSITORY_UUID, branchPath.getPath())
+				.execute(bus)
+				.getSync();
```

Comment:
Revert all changes, and instead of using the predefined language preferences by the server, use the correct locales

Reply:
since the setLocales does not effect the setExpand anyway (as discussed with @cmark ), I removed it.

Reply:
According to our conversation, I said that the current expansion parameter `preferredDescriptions()` does not use locales because you are expanding all preferred descriptions of the concepts and not a single specific description. I also said that you should use either `pt()` or `fsn()` expansion with the client's locales to reuse logic and not reinvent it (custom description lookup code can be removed). 
@norban suggested this in his review comment.